packages/botonic-core/src/handoff.js

```diff
@@ -151,3 +161,12 @@ export async function getAvailableAgents(session) {
   })
   return resp.data
 }
+
+export function cancelHandoff(session, typification = null) {
+  let params = { typification }
+  session._botonic_action = `discard_case:${JSON.stringify(params)}`
```

Comment:
when typification is not passed this will "generate discard_case:null". Does backend support this?